src/Web/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM mcr.microsoft.com/dotnet/sdk:7.0 AS BUILD
+COPY ["./src", "/src"]
+WORKDIR /src/Web
+RUN rm *.db
```

Comment:
`RUN if test -n "$(find . -maxdepth 1 -name '*.db' -print -quit)"; then rm *.db; fi `

when no db is given then the dockerfile will fail

Reply:
Thanks for the finding, fixed it.